partner_classification/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024 Trobz
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Partner Classification",
+    "summary": "This module is used to classify partners",
+    "version": "17.0.1.0.0",
+    "category": "Sales",
+    "author": "Trobz, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": [
+        "base",
+        "sale_management",
```

Comment:
why this dependency ?

Reply:
Hello @legalsylvain, Thanks for your review,
I have created a submenu in the sale menu so the depend here is necessary

Reply:
I see the new menus quite unrelated to the feature you propose. Don't you think ?

Reply:
You're right, I think it would be more appropriate for it to belong to contacts.